guide/14-deep-learning/leveraging_llms_for_real_time_decision_making_in_calamities.ipynb

```diff
@@ -0,0 +1,1029 @@
+{
```

Comment:
<p>Highlight the class names-&nbsp;Health Emergency, Public Service Failures, and Other.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Done</p>